### PR TITLE
Organization page

### PR DESCRIPTION
**Pull request by QaseemLodhi**

### CODE REVIEW

File: client/src/containers/Organisation/Organisation.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+};
+
+class Organisation extends Component<Props> {
+  constructor(props) {
+    super(props);
+    this.state = { organisation: {} };
+  }
+  componentDidMount() {
+    this.getOrganisationDetail();
+    const mockOrganisation = {
```
Comment (mashhoodr) [inline client/src/containers/Organisation/Organisation.js]: @QaseemLodhi why are we still using mock data?

File: client/src/reducers/index.js [hunk 2]
```diff
@@ -90,11 +90,39 @@ const events = (state = defaultEventState, action) => {
...
+  return state;
+};
+
+const defaultOrganisationState = {
+  organisations: [],
+};
+
+const organisationDetail = (state = defaultOrganisationState, action) => {
+  if (action.organisation) {
+    console.log(
```
Comment (mashhoodr) [inline client/src/reducers/index.js]: please dont commit logs

File: client/src/services/organisation.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+const baseUri = '/api/organisations/';
+
+export default {
+  getAll: () => {
+    return makeRequest(baseUri);
+  },
+
+  getOrganisationDetail: organisationId => {
+    return makeRequest(baseUri + organisationId + '/');
```
Comment (mashhoodr) [inline client/src/services/organisation.js]: use template literals

File: client/src/containers/Organisation/Organisation.js (hunk 1 above)
Comment (QaseemLodhi) [reply client/src/containers/Organisation/Organisation.js]: some field were not in api object but I will resolve it asap.

File: client/src/services/organisation.js (hunk 3 above)
Comment (QaseemLodhi) [reply client/src/services/organisation.js]: Ok.

**Pull request closed without merging**